eth/test/MostL2.js

```diff
@@ -180,9 +166,7 @@ describe("MostL2", function () {
     });
 
     it("Emits correct event", async () => {
-      const { most, bazeroAdrressBytes } = await loadFixture(
-        deployEightGuardianMostFixture,
-      );
+      const { most } = await loadFixture(deployEightGuardianMostFixture);
```

Comment:
in `deployEightGuardianMostFixture`, we still call `await most.addPair(bazeroAdrressBytes, bazeroAdrressBytes, false);`, but we don't do it anymore in `eth/scripts/1_l2_setup_bridge_contracts.js`. So not sure which version is correct

Reply:
The 1_l2_* is correct. Fixed

Reply:
Do we need `bazeroAdrressBytes` at all returned by `deployEightGuardianMostFixture`? I mean in those tests, we call `receiveRequest` always with `bazeroAdrressBytes` as `destTokenAddress` argument. I mean, those tests pass right, but do they test native transfers?

Reply:
Good catch again. Sorry I was doing it too fast :P. Anyway, fixed the tests (see the usage of abs)